comp/core/workloadmeta/workloadmeta.go

```diff
@@ -7,13 +7,15 @@ package workloadmeta
 
 import (
 	"context"
-	flaretypes "github.com/DataDog/datadog-agent/comp/core/flare/types"
 	"sync"
 	"time"
 
+	flaretypes "github.com/DataDog/datadog-agent/comp/core/flare/types"
+
```

Comment:
is it expected this space between internal dependencies? 

```suggestion
	flaretypes "github.com/DataDog/datadog-agent/comp/core/flare/types"
```

Reply:
no, fixed :+1: